### PR TITLE
Adds fixture data for many services' articles

### LINKED ISSUE #2486: Add Optimo fixture data for all world service sites
**Is your feature request related to a problem? Please describe.**
We need to generate Optimo fixture data fro all world service sites. This should be limited to one fixture file per service for now.

**Describe the solution you'd like**
For now we can probably copy and paste the /news Optimo fixture data and simply find and replace search and replace `News`, `news` and `en-gb` for each service

**Describe alternatives you've considered**
Write a custom optimo article for each world service site.

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**

### PR DESCRIPTION
**Pull request by jimjohnsonrollings**
Resolves #2486 

**Overall change:**

Adds new fixture data articles for more services

**Code changes:**

* Creates folders for new services
* Adds an article fixture for each
  * Two for each of the script switcher services
  * Two for Ukrainian (one in Russian)
  * Two for Uzbek (one Latin, one Cyrilic)
* Updates tests to take these into account

---

** Testing notes **
* The assets for Serbian, Zhongwen and UK China will likely not yet be requestable over their URLs as we haven't done any work to handle the script switching nature of these services.
* Sport won't work because we haven't configured that services
* Brasil/Portuguese won't work because we currently don't handle service names that are different from their URLs

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Comment by jamesbhobbs** [comment]
Tester should check each article is viewable locally without errors

**Review by jamesbhobbs — APPROVED** [review]

**Review by dr3 — APPROVED** [review]
LGTM, makes assumptions about how we do simp/trad but easy to adjust if we change our minds

File: dataValidator/helpers/dataLoader/asyncValidateDir.test.js [hunk 1]
```diff
@@ -33,13 +33,13 @@ describe('asyncValidateDir helper', () => {
   it('should call readScenario for every file in the /data directory', async () => {
     const readScenarioSpy = jest.spyOn(readScenario, 'readScenario');
 
-    await expectMethodToBeCalledTimes(140, readScenarioSpy);
+    await expectMethodToBeCalledTimes(231, readScenarioSpy);
```
Comment (jamesbhobbs) [inline dataValidator/helpers/dataLoader/asyncValidateDir.test.js]:
```suggestion
    await expectMethodToBeCalledTimes(236, readScenarioSpy);
```

File: dataValidator/helpers/dataLoader/asyncValidateDir.test.js [hunk 2]
```diff
@@ -33,13 +33,13 @@ describe('asyncValidateDir helper', () => {
...
   });
 
   it('should call fileToValidate for only the valid json file in the /data directory', async () => {
     fileToValidateSpy = jest.spyOn(readScenario, 'fileToValidate');
 
-    await expectMethodToBeCalledTimes(42, fileToValidateSpy);
+    await expectMethodToBeCalledTimes(88, fileToValidateSpy);
```
Comment (jamesbhobbs) [inline dataValidator/helpers/dataLoader/asyncValidateDir.test.js]:
```suggestion
    await expectMethodToBeCalledTimes(90, fileToValidateSpy);
```

**Pull request merged**